apps/graph/graph/graph_view.cpp

```diff
@@ -19,6 +19,7 @@ void GraphView::reload() {
     KDRect dirtyZone(KDRect(0, 0, bounds().width(), bounds().height()-m_bannerView->bounds().height()));
     markRectAsDirty(dirtyZone);
   }
+  markRectAsDirty(bounds());
```

Comment:
Why are you not using the AppsContainer reload function ?

Reply:
When you use here the AppsContainer reload function, the app just crash when you enter in the graph mode ...